test/functional/ui/045_folding_spec.lua

```diff
@@ -0,0 +1,218 @@
+-- Tests for folding.
+local Screen = require('test.functional.ui.screen')
+
+local helpers = require('test.functional.helpers')(after_each)
+local feed, insert, execute, expect =
+  helpers.feed, helpers.insert, helpers.execute, helpers.expect
+
+local session = helpers
+
+describe('folding', function()
+  local screen
+
+  before_each(function()
+    session.clear()
+
+    screen = Screen.new(20, 8)
+    screen:attach()
+  end)
+  after_each(function()
+    screen:detach()
+  end)
+
+  it('Test creation, opening, moving to the end and close', function()
```

Comment:
don't need the word "test", it's implied. Also note this description is concatenated with the containing `describe()`, so it reads:

> folding ~~Test~~ creation, opening, moving to the end and close